test/features/support/data_statistics.js

```diff
@@ -1,105 +1,95 @@
-const pactum = require('pactum');
+const chai = require('chai');
+const { spec } = require('pactum');
 const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
-const { localhost, header } = require('./helpers/helpers');
+const {
+  localhost,
+  dataStatisticsEndpoint,
+  dataStatisticsResponseSchema,
+  defaultExpectedResponseTime,
+} = require('./helpers/helpers');
+
+chai.use(require('chai-json-schema'));
 
 let specDataStatistics;
-let startDate;
-let endDate;
 
-const baseUrl = `${localhost}data/statistics/1.0`;
+const baseUrl = localhost + dataStatisticsEndpoint;
 
-Before(() => {
-  specDataStatistics = pactum.spec();
+Before({ tags: '@endpoint=/data/statistics/1.0' }, () => {
```

Comment:
`{ tags: '@endpoint=/data/statistics/1.0' }` is used twice in this file, so better is to move it to constant, e.g.:
const ... = { tags: '@endpoint=/data/statistics/1.0' }
or 
const ... = { tags: '@endpoint=${dataStatisticsEndpoint}' }

Reply:
I wonder if it's possible to use constant `dataStatisticsEndpoint` from `helpers.js` file in `.feature` file

Reply:
Yes it is, good idea. It'd look like ```const ... = { tags: `endpoint=${dataStatisticsEndpoint}` };```